cmd/tnf/addclaim/addclaim.go

```diff
@@ -0,0 +1,113 @@
+package addclaim
+
```

Comment:
I think the package can be claim instead of addclaim. We can add more functions to this package such as update/modify the claim, so it's better to keep it generic